Hymns/Display/Song Info/SongInfoDialogViewModel.swift

```diff
@@ -23,8 +23,10 @@ class SongInfoDialogViewModel: ObservableObject {
         self.identifier = identifier
         self.mainQueue = mainQueue
         self.repository = repository
+        fetchSongInfo()
```

Comment:
This is a good idea. Should we be doing this on some of our other view models instead of using the view with on appear?

Reply:
No. I only did this because we need to know ahead of time whether or not the song info list is populated or not so we know whether or not to show the song info button. But in general, we shouldn't make the request until onAppear.